AssignmentOne/AssignmentOne/Array/Commonele.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AssignmentOne.Array
+{
+    internal class Commonele
+    {
+        public void commonelement()
+        {
+            int[] arr = { 1, 2, 3, 4, 5, };
+            int[] arr2 = { 1, 2, 3, 4, };
+          
+            for(int i =0; i < arr.Length; i++)
+            {
+                for(int j=0;j< arr2.Length; j++)
+                {
+                    if (arr[i] == arr[j]) { 
+                    
+                    Console.WriteLine( "CommonElements" +arr[i]);
```

Comment:
Please print the output in single line separated by comma ','. You can use empty string to store the output and print the final output at once.